config/environments/default.py

```diff
@@ -1,18 +1,17 @@
 """Flask configuration."""
 import logging
 from os import environ
-from os import path
-from fsd_tech import configclass
+from pathlib import Path
+
+from fsd_utils import configclass
 
 
 @configclass
-class Config(object):
+class DefaultConfig(object):
```

Comment:
This is the wrong time to raise this - but it's just occurred to me we're missing a trick here in that the DefaultConfig should really extend a 'ServiceConfig' class in fsd_utils with all the api urls in it so these can be centrally configured.

Reply:
That is some great thinking! I'll add it to my to-do's :)